src/wallet/rpcwallet.cpp

```diff
@@ -843,8 +845,8 @@ static UniValue sendmany(const JSONRPCRequest& request)
             "       \"CONSERVATIVE\""},
                 },
                  RPCResult{
-            "\"txid\"                   (string) The transaction id for the send. Only 1 transaction is created regardless of \n"
-            "                                    the number of addresses.\n"
+                     RPCResult::Type::STR, "txid", "The transaction id for the send. Only 1 transaction is created regardless of\n"
```

Comment:
Can this be of type HEX_STR? 

I'm still new to Bitcoin, my current understanding is that transactions are represented in hex.

Reply:
Thanks. Fixed in 6817c3af8ca13153bf5f85be26eb1db3845b8cac